content/hardware/02.hero/boards/leonardo/features.md

```diff
@@ -1,31 +1,20 @@
 <FeatureDescription>
+
 The Arduino Leonardo is a microcontroller board based on the ATmega32u4. It has 20 digital input/output pins (of which 7 can be used as PWM outputs and 12 as analog inputs), a 16 MHz crystal oscillator, a micro USB connection, a power jack, an ICSP header, and a reset button. It contains everything needed to support the microcontroller; simply connect it to a computer with a USB cable or power it with a AC-to-DC adapter or battery to get started.
+
 </FeatureDescription>
 
 <FeatureList>
 
-<Feature title="ATmega32U4" image="core">
-
-The high Performance, low power AVR® 8-bit microcontroller.
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7766-8-bit-AVR-ATmega16U4-32U4_Datasheet.pdf" download blank/>
-</Feature>
-
-<Feature title="Built-in USB communication" image="usb">
+<Feature title="USB communication" image="usb">
 
-The ATmega32U4 has built-in USB communication that allows the Micro to appear as a mouse/keyboard on your machine.
+The Arduino Leonardo has built-in USB communication that allows the Micro to appear as a mouse/keyboard on your machine.
```

Comment:
Can you add the links to the mouse/keyboard docs?
It is here:

- https://www.arduino.cc/reference/en/language/functions/usb/keyboard/
- https://www.arduino.cc/reference/en/language/functions/usb/mouse/

Reply:
Added both as secondary buttons